src/main/java/com/simibubi/create/content/logistics/filter/ItemAttribute.java

```diff
@@ -273,6 +277,55 @@ public ItemAttribute readNBT(CompoundTag nbt) {
 
 	}
 
+	public static class InItemGroup implements ItemAttribute {
+		public static final InItemGroup EMPTY = new InItemGroup(null);
+
+		private CreativeModeTab group;
+
+		public InItemGroup(CreativeModeTab group) {
+			this.group = group;
+		}
+
+		@Override
+		public boolean appliesTo(ItemStack stack) {
+			return group != null && group.contains(stack);
+		}
+
+		@Override
+		public List<ItemAttribute> listAttributesOf(ItemStack stack) {
+			return CreativeModeTabs.tabs().stream()
```

Comment:
.stream should not be used here, use a enhanced for loop as streams can have performance issues